docs/api/context-bridge.md

```diff
@@ -61,11 +61,17 @@ The `contextBridge` module has the following methods:
 * `apiKey` string - The key to inject the API onto `window` with.  The API will be accessible on `window[apiKey]`.
 * `api` any - Your API, more information on what this API can be and how it works is available below.
 
-### `contextBridge.evaluateInMainWorld(code)` _Experimental_
+### `contextBridge.evaluateInMainWorld(evaluationScript)` _Experimental_
```

Comment:
Should this be called `contextBridge.executeInMainWorld` instead?  Using `evaluate` in the name makes it seem like we are using `eval()`

Reply:
Initially I was using [ShadowRealm.prototype.evaluate](https://tc39.es/proposal-shadowrealm/#sec-shadowrealm.prototype.evaluate) as the inspiration for naming.

I'm open to changing this. Especially since it's been revised to accept func/arg params.

Maybe executeScriptInMainWorld?

Reply:
@samuelmaddock I think that `executeScriptInMainWorld` is fine but a bit long but I don't think its a deal breaker either way.

Reply:
Renamed in [d014f5c](https://github.com/electron/electron/pull/44411/commits/d014f5c06a602b9f6b62fae118a0f886cbbdb202)